Dockerfile

```diff
@@ -28,6 +33,8 @@ COPY public /srv/ttn-lorawan/public
 
 COPY --from=builder /srv/ttn-lorawan/lorawan-frequency-plans /srv/ttn-lorawan/lorawan-frequency-plans
 COPY --from=builder /srv/ttn-lorawan/lorawan-webhook-templates /srv/ttn-lorawan/lorawan-webhook-templates
+COPY --from=builder /srv/ttn-lorawan/device-repository /srv/ttn-lorawan/device-repository
+RUN chown thethings:thethings -R /srv/ttn-lorawan/device-repository
```

Comment:
Is this necessary? The Things Stack only needs to read these files, right?

Reply:
It is necessary for bleve.